### PR TITLE
Add Kashyap to the AUTHORS.md file

### PR DESCRIPTION
**Pull request by kytrinyx**
:tada: :heart: :balloon: :clap:

### CODE REVIEW

**Comment by zzak** [comment]
<3 <3 <3 <3

File: AUTHORS.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Sinatra was designed and developed by Blake Mizerany in California.
 
 * **Konstantin Haase** (maintainer)
 * **Zachary Scott**
-* **Katrina Owen**
```
Comment (rkh) [inline AUTHORS.md]: :( <3 thank you for all the great work

File: AUTHORS.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Sinatra was designed and developed by Blake Mizerany in California.
...
+* **Kashyap Kondamudi**
```
Comment (rkh) [inline AUTHORS.md]: :) <3 thank you for all the great work

File: AUTHORS.md (hunk 1 above)
Comment (kytrinyx) [inline AUTHORS.md]: <3

**Pull request merged**